lib/main.dart

```diff
@@ -431,24 +432,46 @@ class _CardInterfaceState extends State<CardInterface> {
     return _verseStatus == VerseStatus.read;
   }
 
+  double getTextFieldMaxWidth() {
+    return 120.0;
+  }
+
   Widget recitedOrReadSetting() {
     ValueChanged<bool> onChanged = (v) {
       setState(() {
         _verseStatus = v ? VerseStatus.read : VerseStatus.recited;
       });
     };
     return Row(
-      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      mainAxisAlignment: MainAxisAlignment.center,
       children: [
-        Text("Recited"),
-        Switch(
-          value: _verseStatus == VerseStatus.read,
-          onChanged: onChanged,
-          inactiveTrackColor: Colors.grey[500],
-          activeColor: Colors.white,
-          activeTrackColor: Colors.grey[500],
+        Expanded(
+          child: Container(
+            width: getTextFieldMaxWidth(),
+            child: Text(
+              "Recited",
+              textAlign: TextAlign.center,
+            ),
```

Comment:
You don't need this Container. You also don't need to specify the width because Expanded will expand to fill the space in the row. With two Expandeds, they each will take up the same amount of space (and force its child to expand to that, AFAIK (as far as I know)).

Reply:
so to go even further, because of the Expanded's behavior, we actually don't even need the `MainAxisAlignment.center`, but we can keep that